examples/webgl_physics_terrain.html

```diff
@@ -204,7 +204,7 @@
 
 						// Create the terrain body
 
-						var groundShape = this.createTerrainShape( heightData );
+						var groundShape = createTerrainShape( heightData );
```

Comment:
The parameter of `createTerrainShape()` can be removed.

Reply:
Removed! Thanks!